terraform-zerotrust-ssh-http-gcp/argo.tf

```diff
@@ -0,0 +1,11 @@
+# The random_id resource is used to generate a 35 character secret for the tunnel
+resource "random_id" "argo_secret" {
```

Comment:
I'd refer to it as `tunnel_secret` (as cloudflared logs do) rather than `argo_secret`

Reply:
Good call and I updated this on both resources in `argo.tf` and the reference to the resource in `instance.tf`.